lib/processor.js

```diff
@@ -12,6 +12,9 @@ var assign = require("object-assign");
 var remark = require("remark");
 
 var SUPPORTED_SYNTAXES = ["js", "javascript", "node", "jsx"];
+var UNSATISFIABLE_RULES = [
```

Comment:
A way to apply config to a postprocess step in a plugin. Something like:

```json
{
    "pluginConfig": {
        "markdown": {
            "unsatisfiableRules": ["eol-last", "react/no-multi-comp"]
        }
    }
}
```

I checked on the plugin documentation and didn't see anything like that.